tzlocal/__init__.py

```diff
@@ -1,7 +1,34 @@
 import sys
-if sys.platform == 'win32':
-    from tzlocal.win32 import get_localzone, reload_localzone
-elif 'darwin' in sys.platform:
-    from tzlocal.darwin import get_localzone, reload_localzone
-else:
-    from tzlocal.unix import get_localzone, reload_localzone
+
+__version__ = "1.5dev0"
```

Comment:
We can skip this a __version__ isn't required.

If we must have one, this works:

```
import pkg_resources  # part of setuptools
version = pkg_resources.require("tzlocal")[0].version
```

Reply:
It's a convention